cluster/storage/src/main/java/com/linkedin/openhouse/cluster/storage/local/LocalStorage.java

```diff
@@ -52,17 +51,4 @@ public StorageType.Type getType() {
   public StorageClient<?> getClient() {
     return localStorageClient;
   }
-
-  /**
-   * Allocates Table Space for the Local Storage.
-   *
-   * <p>tableLocation looks like: /{rootPrefix}/{databaseId}/{tableId}-{tableUUID} We strip the
-   * endpoint to ensure backward-compatibility. This override should be removed after resolving <a
-   * href="https://github.com/linkedin/openhouse/issues/121">
-   */
-  @Override
-  public String allocateTableLocation(
```

Comment:
we have to be careful with this change as well. tables-test-fixtures uses non-schemed paths. Please validate tables-test-fixtures continue to work after this change cc: @jiang95-dev